nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -39,24 +39,29 @@ class LazyNeMoIterator(ImitatesDict):
     - "text" (overridable with ``text_field`` argument)
 
     Specially supported keys are:
+    - [recommended] "sampling_rate" allows us to provide a valid Lhotse ``Recording`` object without checking the audio file
     - "offset" for partial recording reads
     - "lang" is mapped to Lhotse superivsion's language (overridable with ``lang_field`` argument)
 
     Every other key found in the manifest will be attached to Lhotse Cut and accessible via ``cut.custom[key]``.
 
     .. caution:: We will perform some I/O (as much as required by soundfile.info) to discover the sampling rate
         of the audio file. If this is not acceptable, convert the manifest to Lhotse format which contains
-        sampling rate info.
+        sampling rate info. For pure metadata iteration purposes we also provide a ``dummy_mode`` flag that
+        will create only partially valid Lhotse objects (with metadata related to sampling rate / num samples missing).
 
     Example::
 
         >>> cuts = lhotse.CutSet(LazyNeMoIterator("nemo_manifests/train.json"))
     """
 
-    def __init__(self, path: str | Path, text_field: str = "text", lang_field: str = "lang") -> None:
+    def __init__(
+        self, path: str | Path, text_field: str = "text", lang_field: str = "lang", dummy_mode: bool = False
+    ) -> None:
         self.source = LazyJsonlIterator(path)
         self.text_field = text_field
         self.lang_field = lang_field
+        self.dummy_mode = dummy_mode
```

Comment:
Dummy mode is a poor name imo. Anything more reasonable?

Reply:
The only thing I can think of is `missing_sampling_rate_ok: bool = False` -- I really wish this option wasn't needed :)